ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -159,40 +161,110 @@ const svGenotype = (genotype, isHemiX) => {
   )
 }
 
-export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) => (
-  <AlleleContainer>
-    {warning && (
-      <Popup
-        wide
-        trigger={<WarningIcon />}
-        content={
-          <div>
-            <b>Warning: </b>
-            {warning}
-          </div>
-        }
-      />
-    )}
-    {variant.svType ? (
-      <Header.Content>
-        {svGenotype(genotype, isHemiX)}
-      </Header.Content>
-    ) : (
-      <Header.Content>
-        <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
-        /
-        {isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
-        {genotype.mitoCn && (copyNumberGenotype(genotype.mitoCn, true))}
-      </Header.Content>
-    )}
-  </AlleleContainer>
-))
+const AllelesHeaderContent = ({ genotypeInfo, variant, useStyledAllele }) => {
+  const renderAllele = (isAlt, textAlign) => {
+    if (useStyledAllele) {
+      return (
+        <Allele
+          isAlt={isAlt}
+          variant={variant}
+          textAlign={textAlign}
+        />
+      )
+    }
+    return isAlt ? variant.alt : variant.ref
+  }
+
+  return (
+    <div>
+      {renderAllele(genotypeInfo.genotype.numAlt > (genotypeInfo.isHemiX ? 0 : 1), 'right')}
+      /
+      {genotypeInfo.isHemiX ? '-' : renderAllele(genotypeInfo.genotype.numAlt > 0, 'left')}
+      {genotypeInfo.genotype.mitoCn && copyNumberGenotype(genotypeInfo.genotype.mitoCn, true)}
+    </div>
+  )
+}
+
+AllelesHeaderContent.propTypes = {
+  genotypeInfo: PropTypes.object,
+  variant: PropTypes.object,
+  useStyledAllele: PropTypes.bool,
+}
+
+AllelesHeaderContent.defaultProps = { useStyledAllele: true }
+
+export const Alleles = React.memo(({ variant, genotypes }) => {
```

Comment:
This class is exported and used elsewhere and I do not think we want to change the behavior for everywhere it is used, as much of this new logic is not relevant. For example, the fact that this component could take any "warning" but now explicitly computes warning is a loss of functionality for the component. 

Keep the exported `Alleles` class the same as much as possible. The allele display itself should not change, but if there is anything you really need to change then add a nw prop to the class to conditionally change that. Then add a `MultiSampleTypeAlleles` or a `DetailedAlleles` component to change the behavior that needs to change just within this component

Reply:
These are great points, I added a new `MultiSampleTypeAlleles` component instead of modifying the original `Alleles` to support this case